src/nxt_tstr.c

```diff
@@ -80,16 +80,17 @@ nxt_tstr_compile(nxt_tstr_state_t *state, const nxt_str_t *str,
 {
     u_char      *p;
     nxt_tstr_t  *tstr;
-    nxt_bool_t  strz;
+    nxt_bool_t  strz, newline;
 
     strz = (flags & NXT_TSTR_STRZ) != 0;
+    newline = (flags & NXT_TSTR_NEWLINE) != 0;
```

Comment:
I don't know why we do the test like this (!= 0) it can _only_ be either set or not.

We don't do it like that in the if () statements...

Reply:
Makes sense.

Reply:
Yes, indeed... sorry brain fart on my part...

Reply:
No worries, tests can help in this case.